feature-configs/hypershift-ci/localhost-workaround/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../deploy/localhost-workaround
```

Comment:
please check this file I think it use a MCP

Reply:
yep, removing